app/models/concerns/github_handle.rb

```diff
@@ -1,15 +0,0 @@
-module GithubHandle
-  extend ActiveSupport::Concern
-
-  def github_handle
-    user.try(:github_handle)
-  end
-
-  def github_handle=(github_handle)
-    return unless github_handle.present?
-    self.user = User.where('github_handle ILIKE ?', github_handle)
-                    .first_or_initialize(github_handle: github_handle)
-    user.github_handle = github_handle
```

Comment:
I removed this line. At this point, every user instance should have the `github_handle` set properly already. Or am I overlooking something?

Reply:
Yes, `first_or_initialize` should have done it already 👍